SimGeneral/MixingModule/python/digitizers_cfi.py

```diff
@@ -101,6 +101,7 @@
 
 theDigitizersValid = cms.PSet(theDigitizers)
 theDigitizers.mergedtruth.select.signalOnlyTP = True
+theDigitizersValid.calotruth = cms.PSet( caloParticles )
```

Comment:
Shouldn't this also be part of an era-modifier?

Reply:
@rovere you are right! Just changed it